PlaygroundService/Controllers/PlaygroundController.cs

```diff
@@ -22,6 +23,24 @@ public PlaygroundController(IClusterClient client, ILogger<PlaygroundController>
             _client = client;
             _logger = logger;
         }
+        
+        [HttpGet("templates")]
+        public async Task<IActionResult> GetTemplateConfig()
+        {
+            _logger.LogInformation("templates  - GetTemplateConfig started time: "+ DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+            List<string> myList = new List<string> { "item1", "item2", "item3" };
+
+            return Ok(myList); 
+        }
+        
+        [HttpGet("templateInfo")]
+        public async Task<IActionResult> GetTemplateInfo([FromQuery] string template, [FromQuery] string templateName)
```

Comment:
`templateName ` rename to `projectName` might be more intuitive.

Reply:
done